keepsorted/options_test.go

```diff
@@ -100,6 +118,15 @@ func TestBlockOptions(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "ItemSet_YAML",
+			in:   `allow_yaml_lists=yes sticky_prefixes=[a, b, c d, 'e",\f', 'g]', "h\"]"]`,
+
+			want: blockOptions{
+				AllowYAMLLists: true,
+				StickyPrefixes: map[string]bool{"a": true, "b": true, "c d": true, `e",\f`: true, "g]": true, `h"]`: true},
+			},
+		},
```

Comment:
As mentioned above, we seem to be missing the notable test case of params existing after a YAML list (matters since it's important that the YAML parsing correctly terminated and left the rest of the options in a parsable state).